x/evm/genesis_test.go

```diff
@@ -45,9 +44,7 @@ var _ = g.Describe("genesis", func() {
 	g.BeforeEach(func() {
 		t := g.GinkgoT()
 		a = app.NewTestApp(t, false)
-		ctx = a.NewContext(false, tmproto.Header{
-			Height: 5,
-		})
+		ctx = a.NewContext(false)
```

Comment:
Where are we setting starting to 5?

Reply:
NewContext function takes only one parameter i.e bool  and in NewContext function it calls the  other  function with the empty header.
In order to set the starting to 5, we need to use NewUncachedContext .